README.md

```diff
@@ -1,38 +1,81 @@
 # DeepSlide: A Sliding Window Framework for Classification of High Resolution Microscopy Images (Whole-Slide Images)
 
-This repository is a sliding window framework for classification of high resolution whole-slide images, often called microscopy or histopathology images. This is also the code for the paper [Pathologist-level Classification of Histologic Patterns on Resected Lung Adenocarcinoma Slides with Deep Neural Networks](https://www.nature.com/articles/s41598-019-40041-7). For a practical guide and implementation tips, see the medium post [Classification of Histopathology Images with Deep Learning: A Practical Guide](https://medium.com/health-data-science/classification-of-histopathology-images-with-deep-learning-a-practical-guide-2e3ffd6d59c5). 
+This repository is a sliding window framework for classification of high resolution whole-slide images, often called microscopy or histopathology images. 
+This is also the code for the paper 
+[Pathologist-level Classification of Histologic Patterns on Resected Lung Adenocarcinoma Slides with Deep Neural Networks](https://www.nature.com/articles/s41598-019-40041-7).
+ For a practical guide and implementation tips, see the Medium post 
+ [Classification of Histopathology Images with Deep Learning: A Practical Guide](https://medium.com/health-data-science/classification-of-histopathology-images-with-deep-learning-a-practical-guide-2e3ffd6d59c5). 
 
 
 ![alt text](figures/figure-2-color.jpeg)
 
-## Dependencies
-- [Python 3.6](https://www.anaconda.com/download/#macos)
-- [PyTorch](https://pytorch.org/)
-- [PIL](https://pillow.readthedocs.io/en/5.3.x/)
+## Changes in this Release
+- Use Python [multiprocessing](https://docs.python.org/3.6/library/multiprocessing.html)
+to speed up patch creation.
+- Replace [os](https://docs.python.org/3.6/library/os.html) with
+[pathlib](https://docs.python.org/3.6/library/pathlib.html) module.
+- Added [argparse](https://docs.python.org/3.6/library/argparse.html) support.
+- Rewrote patch processing code using vector operations.
+- Fully documented all functions and methods with docstrings and type hints.
+- Compute image means and standard deviations for normalization online.
+- Increased efficiency of code by rewriting inefficient code.
+
+## Known Issues and Limitations
+- Patch creation code doesn't find any patches in images that are slightly larger
```

Comment:
Not clear to me what this means

Reply:
@jasonwei20 When running the processing code on images that were around 300x300 pixels, no patches were found even though we probably should find some.

Reply:
Can you show me an example? Many of the patches that I used were small and they all worked.

Reply:
@jasonwei20 I realized my problem.  I used an overlap factor of 5 even though I should have used 3.